language/stdlib/transaction_scripts/add_recovery_rotation_capability.move

```diff
@@ -2,9 +2,25 @@ script {
 use 0x1::LibraAccount;
 use 0x1::RecoveryAddress;
 
-/// Add the `KeyRotationCapability` for `to_recover_account` to the `RecoveryAddress` resource under `recovery_address`.
-///
-/// ## Aborts
+/// # Sender Preconditions
+/// The sending account (`to_recover_account`) must not have previously given away its key
+/// rotation capability, and must be a VASP account. The account at `recovery_address`
+/// must also be a VASP account belonging to the same VASP as the `to_recover_account`.
+/// Additionally the account at `recovery_address` must have already initalized itself as
+/// a recovery account address using the `create_recovery_address` transaction script.
```

Comment:
Is it possible to make this hyperlink to the `create_recovery_address` script? I see that the doc generator does this automatically for types/functions/modules; not sure if there is or easily could be a way to do it with scripts. CC @wrwg

Reply:
That would be awesome! Also just as a note to myself: I am realizing that constants aren't linking properly, so I will need to go in and fixing that at some point this week.